packages/asset-service/src/generateAssetData/setColors.ts

```diff
@@ -5,7 +5,11 @@ import { Asset } from '../service/AssetService'
 const toHex = (num: number): string => num.toString(16).toUpperCase().padStart(2, '0')
 
 export const setColors = async (assets: Asset[]): Promise<Asset[]> => {
-  for await (const [index, asset] of assets.entries()) {
+  const filteredAssets = assets.filter(
+    (asset) => asset.color === '#FFFFFF' && asset.icon && !asset.icon.includes('yearn'),
```

Comment:
Let's comment out `yearnVaults` instead in a separate PR, we basically disabled Yearn everywhere else

Reply:
https://github.com/shapeshift/lib/pull/1197/commits/43253b260f095c2be7b36ffe3f1c3184f25c44e3